### ISSUE TITLE
Question: ToC per page, selectively

### ISSUE THREAD

**Issue by nazikus** [hybridgroup/GitHub-Wikifier#12]
I'm looking for a way to generate ToC selectively for some pages only. I don't need a sidebar or a separate page for table of contents. Just for some big pages I'd like to generate ToC on top of the page based on headers. This is a great example of what I'm looking for - [wiki: Table Parser for WET 3.0](https://github.com/wet-boew/wet-boew/wiki/Table-Parser-for-WET-3.0)

I have seen [artoo wiki](https://github.com/hybridgroup/artoo/wiki) example, which is great. But is it possible to make ToC  without sidebar and within the page?
Basically I need native Gollum [[_TOC_]] tag ([Gollum wiki](https://github.com/gollum/gollum/wiki#table-of-contents)), since it is not enabled in GitHub be default.
Thanks

**Comment by solojavier** [comment]
Hi @nazikus, thanks for writting, there is currently no option to generate the table of contents on demand. 

I working on refactoring the code and I'll see if I can extract what you are looking for... also I am marking the issue as enhancement so we can keep track.

Regards

**Comment by solojavier** [comment]
Im closing this issue as we are not intending to work on this feature anytime soon, but will be happy to accept pull request to extract this behaviour...
